React/Profiler/RCTPerfMonitor.m

```diff
@@ -158,17 +158,20 @@ - (RCTDevMenuItem *)devMenuItem
   if (!_devMenuItem) {
     __weak __typeof__(self) weakSelf = self;
```

Comment:
nit: make this `__weak RCTDevSettings devSettings = self.bridge.devSettings;` to simplify the code below.